ui/src/app/components/runs/filters/string-filter/string-filter.component.ts

```diff
@@ -69,7 +67,7 @@ export class StringFilterComponent implements ClrDatagridFilterInterface<DagRunM
 
   onRemoveFilter() {
     this.value = undefined;
-    this.modelChanges.next(undefined)
+    this.modelChanges.next(this.value)
```

Comment:
Why is this change needed?

Reply:
No reason. I just didnt want to repeat value.

Reply:
Oh sorry, I completely missed `this.value = undefined;`